src/components/backend-ai-agent-list.ts

```diff
@@ -1225,7 +1225,7 @@ export default class BackendAIAgentList extends BackendAIPage {
         cpu_util: { capacity: 0, current: 0, ratio: 0 },
         mem_util: { capacity: 0, current: 0, ratio: 0 },
       };
-      if (rowData.item.live_stat.node.cuda_util) {
+      if (rowData.item.live_stat && rowData.item.live_stat.node.cuda_util) {
```

Comment:
```suggestion
      if (rowData.item.live_stat?.node?.cuda_util) {
```
How about this? @lizable